pkg/util/mount/mount.go

```diff
@@ -130,20 +131,40 @@ func GetMountRefs(mounter Interface, mountPath string) ([]string, error) {
 		}
 	}
 
+	// TODO: this is a workaround for the unmount device issue caused by gci mounter.
+	// In GCI cluster, if gci mounter is used for mounting, the container started by mounter
+	// script will cause additional mounts created in the container. Since these mounts are
+	// irrelavant to the original mounts, they should be not considered when checking the
+	// mount references. By comparing the mount path prefix, those additional mounts can be
+	// filtered out.
+	// Plan to work on better approach to solve this issue.
+	podsDirPrefix := getPodsDirPrefix(mountPath, options.DefaultKubeletPluginsDirName)
 	// Find all references to the device.
 	var refs []string
 	if deviceName == "" {
 		glog.Warningf("could not determine device for path: %q", mountPath)
 	} else {
 		for i := range mps {
 			if mps[i].Device == deviceName && mps[i].Path != slTarget {
-				refs = append(refs, mps[i].Path)
+				mpPrefix := getPodsDirPrefix(mps[i].Path, options.DefaultKubeletPodsDirName)
+				if mpPrefix == podsDirPrefix {
```

Comment:
My worry with this is that it may skip checks on non-rkt mounts as well. Let's see if we can minimize this.